app/script/conversation/ConversationRepository.coffee

```diff
@@ -1522,7 +1523,11 @@ class z.conversation.ConversationRepository
         else
           @logger.log 'Unsupported ephemeral type', message_et.type
     else
-      @delete_message_everyone conversation_et, message_et
+      if conversation_et.is_group()
+        user_ids = _.union [@user_repository.self().id], [message_et.from]
+        @delete_message_everyone conversation_et, message_et, user_ids
```

Comment:
Should optimally be sent to the sending client of the sender and not all of his clients.